File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/Teacher.java
package com.bnuz.oke.entity;

import java.io.Serializable;

/**
 * Teacher
 *
 * @author handsome
 * @date 2020年 05月14日 23:58:10
 */
public class Teacher implements Serializable {

	private static final long serialVersionUID = -194261173474839551L;

	private int teacherId;

	private String teacherName;

	private String teacherTitle;

	private User user;

	public int getTeacherId() {
		return teacherId;
	}

	public void setTeacherId(int teacherId) {
		this.teacherId = teacherId;
	}

	public String getTeacherName() {
		return teacherName;
	}

	public void setTeacherName(String teacherName) {
		this.teacherName = teacherName;
	}

	public String getTeacherTitle() {
		return teacherTitle;
	}

	public void setTeacherTitle(String teacherTitle) {
		this.teacherTitle = teacherTitle;
	}

	public User getUser() {
		return user;
	}

	public void setUser(User user) {
		this.user = user;
	}

	@Override
	public String toString() {
		return "Teacher{" +
				"teacherId=" + teacherId +
				", teacherName='" + teacherName + '\'' +
				", teacherTitle='" + teacherTitle + '\'' +
				", user=" + user +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/Teacher.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 老师信息封装类
    /// </summary>
    [Serializable]
    public class Teacher
    {
        //老师id
        public int teacherId { set; get; }
        //老师姓名
        public string teacherName { set; get; }
        //老师职称
        public string teacherTitle { set; get; }
        //老师账号
        public User user { set; get; }
    }
}
